tools/codegen/model.py

```diff
@@ -1739,6 +1802,95 @@ def gets_generated_out_inplace_wrapper(f: NativeFunction, g: NativeFunctionsGrou
         not b.has_kernel(f) and \
         b.has_kernel(g.functional)
 
+# NativeFunction objects that are views (f.is_view_op returns True)
+# are added into a `NativeFunctionsViewGroup`, which we can use to
+# easily access the generated (optional) view_copy NativeFunction.
+# It's convenient to group them together, so we pair them up in NativeFunctionsViewGroup.
+# See Note [Codegen'd {view}_copy Operators]
+#
+# One property of this representation is that in order for a view-like op to be part of
+# a NativeFunctionsViewGroup, the "aliasing" version of that view op must exist.
+# There's one case where that doesn't happen: we have a non-aliasing `narrow_copy.out` op,
+# but don't have corresponding aliasing `narrow.out` op.
```

Comment:
why not?!

Reply:
An out= flavor of a view op seems pretty weird to me. out= ops help you avoid allocating new memory, but that's also true for view ops.

That made me lean towards not adding a `narrow.out` (and `narrow_copy.out` not getting grouped into a `NativeFunctionsViewGroup` won't impact the functionalization codegen, since we shouldn't actually end up using that op anywhere)

Reply:
I guess the converse would be if we can get rid of the out variants :)